File: 30-days-of-code/linked-list/solution.cpp
Node* insert(Node *head,int data)
{
    if(head ==NULL)
    {
        Node *current = new Node(data);
        return current;
    }
    // Reach the end of the Linked list
    Node *starting_point = head;
    while(head->next != NULL)
        head = head->next;
    // Create new Node
    Node *new_node = new Node(data);
    head ->next = new_node;
    new_node ->next = NULL;
    head = starting_point;
    return head; 
}

File: 30-days-of-code/linked-list/solution.py
    def insert(self,head,data): 
        new_node = Node(data)
        new_node.next = None
        if head is None:
            return new_node
        original_head = head
        while head.next is not None:
            head = head.next
        head.next = new_node
        head = original_head
        return head